service/controller/clusterapi/v30/cloudconfig/cloud_config_test.go

```diff
@@ -1,197 +0,0 @@
-package cloudconfig
```

Comment:
Dropping these tests. We have golden file tests now. They verify that nothing else changed than injecting the labels.